src/pages/AddItemForm.js

```diff
@@ -18,18 +19,65 @@ async function addToDb(item_name, purchase_interval, user_token) {
     console.error('Error adding document: ', e);
   }
 }
-
 const AddItemForm = (props) => {
   const [itemName, setItemName] = useState('');
   const [purchaseInterval, setPurchaseInterval] = useState('7');
+  const [docs, setDocs] = useState([]);
+
+  //on page load, check the db
+  useEffect(() => {
```

Comment:
I think the contents of this function (the DB call) could be moved into the handleSubmit function so that it only runs when the user is ready for the word to be sent.

Reply:
We tried that but got an infinite loop so we moved it back.  We will def give it another shot!